examples/logical_interconnect.rb

```diff
@@ -61,9 +63,66 @@
 # Activate the port monitor service
 oneview_logical_interconnect 'Encl1-LogicalInterconnectGroup1' do
   client my_client
+  port_monitor(
```

Comment:
Also, IF we want to be merging and using both `data` and `port_monitor`, I'd prefer that the examples illustrate that. So no need for one enable/disable with each, I'd be interested in how we'd be using them together.